src/NuGet.Server.Core/Infrastructure/ServerPackageStore.cs

```diff
@@ -103,17 +103,16 @@ public void Remove(string id, SemanticVersion version, bool enableDelisting)
             await Task.Yield();
 
             // Try to create the server package and ignore a bad package if it fails.
-            var serverPackage = CreateServerPackage(package, enableDelisting, token);
+            var serverPackage = CreateServerPackageOrNull(package, enableDelisting);
             if (serverPackage != null)
             {
                 allPackages.Add(serverPackage);
             }
         }
 
-        private ServerPackage CreateServerPackage(
+        private ServerPackage CreateServerPackageOrNull(
             IPackage package,
-            bool enableDelisting,
-            CancellationToken token)
```

Comment:
This `token` removal addresses this follow-up comment: https://github.com/NuGet/NuGet.Server/pull/28#discussion_r95200934.